include/podio/CollectionBase.h

```diff
@@ -8,7 +8,8 @@
 #include <utility>
 #include <vector>
 
-
+#include "podio/ObjectID.h"
+#include "podio/ASCIIWriter.h"
```

Comment:
I think these can be safely removed, right?

Reply:
Yes, they can